tests/test_file_handler.py

```
@@ -0,0 +1,49 @@
+from __future__ import annotations
```

Pylint (3.3.7)

[convention] Missing module docstring

(C0114)